ArduCopter/system.cpp

```diff
@@ -256,12 +256,20 @@ void Copter::update_dynamic_notch()
 
 #if RPM_ENABLED == ENABLED
         case HarmonicNotchDynamicMode::UpdateRPM: // rpm sensor based tracking
-            float rpm;
-            if (rpm_sensor.get_rpm(0, rpm)) {
-                // set the harmonic notch filter frequency from the main rotor rpm
-                ins.update_harmonic_notch_freq_hz(MAX(ref_freq, rpm * ref / 60.0f));
```

Comment:
ref multiplication is done here

Reply:
Okay, I've added that I believe. Why isn't the division by 60 used by the version where the RPMs are averaged and why doesn't the blheli telem version get multipled by ref? Should I add those as well?

Thanks!